module.go

```diff
@@ -211,6 +211,11 @@ func (m *Module) BPFLoadObject() error {
 	return nil
 }
 
+func (m *Module) BPFReplaceExistingObject(obj unsafe.Pointer) error {
```

Comment:
It's just a straightforward assignment, right? It would be nice to check if it's still possible to replace it and error when not. Please also add documentation (a selftest would be awesome ❤️).